python/common/business-registry-model/README.md

```diff
@@ -1,5 +1,31 @@
 # Business Registry Model
 
+
+# Contributing
+
+## Linting
+For now the lint checks must be doing manually. Please run ruff and fix all issues until you get a clean run, signigifed by a `All checks passed!`
+```bash
+ruff --config pyproject.toml check --fix
+```
+
+## Testing
+The test suite uses *testcontainers*, you can run it locally or you can use the _.devcontainer_ service available is a number of IDEs.
+
+This library uses `pytest`. All test should pass.
+Running this should give you a fully passing and cleared run.
+```bash
+pytest tests
```

Comment:
Opening up devcontainer in VSCode on mine I need to run `poetry run pytest` for getting a pytest run (otherwise `E   ModuleNotFoundError: No module named 'sqlalchemy'`)